src/todolistcheckcommand.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module list/todolistcheckedcommand
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command';
+
+const attributeKey = 'todoListChecked';
+
+/**
+ * @extends module:core/command~Command
+ */
+export default class TodoListCheckCommand extends Command {
+	/**
+	 * @inheritDoc
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		/**
+		 * Flag indicating whether the command is active. The command is active when at least one of
+		 * {@link module:engine/model/selection~Selection selected} elements is a todo list item.
```

Comment:
This description looks more like `isEnabled`, which is missing by the way.

Reply:
Fixed.